### PR TITLE
added CALIB_MODE_SET/GET commands (does nothing yet)

### PR DESCRIPTION
**Pull request by mortenjc**
Subtask in preparation  for ECDC 3774

### CODE REVIEW

File: src/efu/test/ParserTest.cpp
Comment (tbukovics) [inline src/efu/test/ParserTest.cpp:1]: Please update the copyright part to the new format and also update the dates.

File: src/efu/Parser.cpp
Comment (tbukovics) [inline src/efu/Parser.cpp:1]: Update the dates in the copryright part

File: src/efu/Parser.cpp [hunk 1]
```diff
@@ -70,6 +70,45 @@ static int stat_get(std::vector<std::string> cmdargs, char *output,
   return Parser::OK;
 }
 
+//=============================================================================
```
Comment (tbukovics) [inline src/efu/Parser.cpp:73]: Would be nice to have a  \brief description about these functions. If you write one, I can generate similar ones for the others with copilot.

File: src/efu/Parser.cpp [hunk 2]
```diff
@@ -70,6 +70,45 @@ static int stat_get(std::vector<std::string> cmdargs, char *output,
...
+    return -Parser::EBADARGS;
+  }
+
+  detector->CalibrationMode = bool(index);
+  LOG(CMD, Sev::Debug, "CALIB_ {}", index);
+  return Parser::OK;
+}
+
+//=============================================================================
+static int calib_mode_get(std::vector<std::string> cmdargs, char *output,
```
Comment (tbukovics) [inline src/efu/Parser.cpp:93]: Same a brief description of this function would be useful

File: utils/efushell/efustatus.py [hunk 3]
```diff
@@ -16,9 +16,9 @@
 metrics = Metrics(args.i, args.p)
 
 print(metrics._get_efu_command("RUNTIMESTATS"))
-
 print(metrics._get_efu_command("VERSION_GET"))
 print(metrics._get_efu_command("DETECTOR_INFO_GET"))
+print(metrics._get_efu_command("CALIB_MODE_GET"))
```
Comment (tbukovics) [inline utils/efushell/efustatus.py:21]: Shouldn't the CALIB_MODE_SET also added here?

File: src/efu/test/ParserTest.cpp [hunk 4]
```diff
@@ -16,9 +16,10 @@ static int dummy_command(std::vector<std::string>, char *, unsigned int *) {
 // clang-format off
 std::vector<std::string> commands {
   "STAT_GET_COUNT",                 "STAT_GET_COUNT 1",
-  "CMD_GET_COUNT",                  "CMD_GET_COUNT 9",
+  "CMD_GET_COUNT",                  "CMD_GET_COUNT 11",
   "STAT_GET 1",                     "STAT_GET dummystat 42",
   "STAT_GET 2",                     "STAT_GET  -1",
+  "CALIB_MODE_GET",                 "CALIB_MODE_GET 0",
   "EXIT",                           "<OK>"
```
Comment (tbukovics) [inline src/efu/test/ParserTest.cpp:23]: Shouldn't the CALIB_MODE_SET added here?

File: src/efu/Parser.cpp (hunk 1 above)
Comment (mortenjc) [reply src/efu/Parser.cpp:73]: ok

File: utils/efushell/efustatus.py (hunk 3 above)
Comment (mortenjc) [reply utils/efushell/efustatus.py:21]: No, this is intentional, I am only interested in getting the status in this command

File: src/efu/test/ParserTest.cpp (hunk 4 above)
Comment (mortenjc) [reply src/efu/test/ParserTest.cpp:23]: That would go in a different test. I am probably not going to implement that

**Review by tbukovics — APPROVED** [review]

**Pull request merged**